src/commands/deploy/nextjs/deploy.ts

```diff
@@ -380,6 +381,16 @@ async function deployFunctions(config: YamlProjectConfiguration, stage?: string)
     return result;
 }
 
+function writeNextConfig() {
+    const cwd = process.cwd();
+    const extension = ["js", "cjs", "mjs", "ts"].find((ext) =>
+        fs.existsSync(path.join(cwd, `next.config.${ext}`)),
+    );
+    if (!extension) {
```

Comment:
this naming is misleading

```suggestion
    const configExists = ["js", "cjs", "mjs", "ts"].find((ext) =>
        fs.existsSync(path.join(cwd, `next.config.${ext}`)),
    );
    if (!configExists) {
```